From 9939fcd5193046a1e7f0d3d0f2da795bd9ee5f27 Mon Sep 17 00:00:00 2001
From: Ryder Belserion <no-reply@ryderbelserion.com>
Date: Fri, 31 Mar 2023 00:31:39 -0400
Subject: [PATCH] Replace their maven publication

---
 build.gradle.kts | 21 +++++++++++++++++++--
 1 file changed, 19 insertions(+), 2 deletions(-)

diff --git a/build.gradle.kts b/build.gradle.kts
index 3b1ae876..d2427b5c 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -154,9 +154,26 @@ bukkit {
 }
 
 publishing {
+    repositories {
+        maven("https://repo.crazycrew.us/api") {
+            name = "crazycrew"
+            // Used for locally publishing.
+            credentials(PasswordCredentials::class)
+
+            credentials {
+                username = System.getenv("REPOSITORY_USERNAME")
+                password = System.getenv("REPOSITORY_PASSWORD")
+            }
+        }
+    }
+
     publications {
-        register<MavenPublication>("maven") {
-            from(components.getByName("java"))
+        create<MavenPublication>("maven") {
+            groupId = rootProject.group.toString()
+            artifactId = "${rootProject.name.toLowerCase()}-api"
+            version = rootProject.version.toString()
+
+            from(components["java"])
         }
     }
 }
-- 
2.40.1.windows.1

