From 6f3b108d9d8745ad561593ebe6e7f10da35eb345 Mon Sep 17 00:00:00 2001
From: Ryder Belserion <no-reply@ryderbelserion.com>
Date: Fri, 9 Jun 2023 18:03:46 -0400
Subject: [PATCH] resolve merge conflict

---
 gradle.properties | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gradle.properties b/gradle.properties
index c3296c73..c3f74184 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1 +1,3 @@
 pluginVersion=1.156.3
+group = io.th0rgal
+name = Oraxen
\ No newline at end of file
-- 
2.40.1.windows.1

